### PR TITLE
WIP: 58 lint error

### PR DESCRIPTION
**Pull request by vsysoev**

### CODE REVIEW

File: src/pygats/recog.py [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+    recognized = pytesseract.image_to_data(img, txt.lang).split('\n')
+    if one_word is False:
+        combine_words_in_lines(recognized)
+    ret_tuple = (-1, -1, -1, -1, False)
+    for line in recognized[1:]:
+        splitted = line.split('\t')
+        if len(splitted) == 12 and splitted[11].find(txt.text) != -1:
+            print(f'Найден текст {splitted[11]}')
+            ret_tuple = (int(splitted[6]), int(splitted[7]), int(
+                splitted[8]), int(splitted[9]), True)
```
Comment (RiMikheev) [inline src/pygats/recog.py:56]: must either put the lower arguments under the arguments of the upper line, or move the arguments of the upper line to the next line

**Review by RiMikheev — COMMENTED** [review]
The code has been reviewed, comments have been left on some sections of the code

File: src/pygats/recog.py [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+    if one_word is False:
+        combine_words_in_lines(recognized)
+    ret_tuple = (-1, -1, -1, -1, False)
+    for line in recognized[1:]:
+        splitted = line.split('\t')
+        if len(splitted) == 12:
+            if splitted[11].find(txt.text) != -1:
+                print("Найден текст " + splitted[11])
+                ret_tuple = (int(splitted[6]), int(splitted[7]), int(
+                    splitted[8]), int(splitted[9]), True)
```
Comment (RiMikheev) [inline src/pygats/recog.py:266]: must either put the lower arguments under the arguments of the upper line, or move the arguments of the upper line to the next line

File: src/pygats/recog.py [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+        new word to it.
+
+    """
+    for i in range(1, len(lines)-1):
+        splitted = lines[i].split('\t')
+        if len(splitted) != 12:
+            return
+        y = int(splitted[7])
+        for j in range(i+1, len(lines)-1):
+            splitted2 = lines[j].split('\t')
```
Comment (ProjectMutilation) [inline src/pygats/recog.py:183]: splitted2 is bad naming

File: src/pygats/recog.py [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+    Returns:
+        (x,y,w,h,text, substring):
+            x (int), y (int): coordinates of top-left point of rectangle where
+               text resides
+            w (int), h (int): width and height of rectangle where text resides
+            text (string): full text which resides in rectangle
+            substring (string): substring found in text
+    """
+    result = []
+    l_search = len(search)
```
Comment (ProjectMutilation) [inline src/pygats/recog.py:328]: search_len is better name in my opinion

File: src/pygats/recog.py [hunk 5]
```diff
@@ -0,0 +1,366 @@
...
+    for item in recognized_list:
+        r = ratio(search, item[4], score_cutoff=0.5)
+        if r > 0.0:
+            result.append(item)
+        else:
+            s = item[4]
```
Comment (ProjectMutilation) [inline src/pygats/recog.py:334]: magic number, name of variable isn't informative

File: src/pygats/recog.py [hunk 6]
```diff
@@ -245,11 +247,11 @@ def find_text(img, txt, skip=0, inv = False, one_word = False):
...
-                                      cv.ADAPTIVE_THRESH_MEAN_C, \
+                                      cv.ADAPTIVE_THRESH_MEAN_C,
                                       cv.THRESH_BINARY_INV, 11, 2)
     else:
         thresh = cv.adaptiveThreshold(img_gray, 255,
-                                      cv.ADAPTIVE_THRESH_MEAN_C, \
+                                      cv.ADAPTIVE_THRESH_MEAN_C,
                                       cv.THRESH_BINARY, 11, 2)
     recognized = pytesseract.image_to_data(thresh, txt.lang).split('\n')
     if one_word is False:
```
[the remark this answers is not in the document]
Comment (vsysoev) [reply src/pygats/recog.py]: Done

File: src/pygats/recog.py [hunk 7]
```diff
@@ -267,10 +269,12 @@ def find_text(img, txt, skip=0, inv = False, one_word = False):
...
             else:
                 if int(splitted[6]) + int(splitted[7]) != 0:
-                    cropped = Image.fromarray(thresh).crop((int(splitted[6]), int(splitted[7]),
-                                        int(splitted[6])+int(splitted[8]),
-                                        int(splitted[7])+int(splitted[9])))
-                    cropped_tuple = find_cropped_text(cropped, txt, 0, one_word)
+                    cropped = Image.fromarray(thresh).crop(
+                        (int(splitted[6]), int(splitted[7]),
+                            int(splitted[6])+int(splitted[8]),
+                            int(splitted[7])+int(splitted[9])))
```
[the remark this answers is not in the document]
Comment (vsysoev) [reply src/pygats/recog.py]: Done

File: src/pygats/recog.py [hunk 8]
```diff
@@ -0,0 +1,360 @@
...
+        (left, top, width, height, found):
+            left (int): left coordinate of the text bounding box
+            top (int): top coordinate of the text bounding box
+            width (int): width of the text bounding box
+            height (int): height of the text bounding box
+            found (bool): whether the text is found in the image
+
+    """
+    recognized = pytesseract.image_to_data(img, txt.lang).split('\n')
+    if one_word is False:
```
[the remark this answers is not in the document]
Comment (vsysoev) [reply src/pygats/recog.py]: Done

File: src/pygats/recog.py (hunk 3 above)
Comment (vsysoev) [reply src/pygats/recog.py]: yep

**Review by ProjectMutilation — APPROVED** [review]

**Pull request merged**